app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -73,7 +73,9 @@
 	double lon;
 	int statusScreen = 0;
 	View view;
-
+    //Dimensions for Camera Switch and Flash Button
+    int width = 60;
```

Comment:
Still the static value. That's not the point of adding them pro-grammatically. It should be screen width and height based.
Add two functions in the `Utils` to calculate screen height and screen width. According to them give the value of width and height of icons.